### ISSUE TITLE
Configuration table is hard to read

### ISSUE THREAD

**Issue by lentzi90** [kubeapps/hub#55]
Several charts use a table with parameter, description and default value columns. Unfortunately, it is very hard to read the tables because of the narrow column used for the content. See [Prometheus](https://hub.kubeapps.com/charts/stable/prometheus) for an example.

I'm not suggesting that we just change the `max-width`. This would make the rest of the text hard to read with too long lines. Maybe there could be a separate section just for the table, with full width?

What do you think?

**Comment by prydonius** [comment]
Agreed @lentzi90, it does look pretty difficult to read. I'm not sure how much control we have over the table rendering since we're just using https://github.com/markedjs/marked. @Angelmmiguel any ideas?

**Comment by Angelmmiguel** [comment]
You're right @prydonius. Since we're rendering the markdown file it's quite complicated to customize the table. The same issue happens on Github: https://github.com/helm/charts/tree/master/stable/prometheus#configuration. 

We can make more obvious you need to scroll the table using an inner shadow. However, that's not gonna solve the readability issue.

Another option could be to include an "expand table" button that removes the `overflow: auto;` property: 

![Table without overflow: auto](https://user-images.githubusercontent.com/4056725/47799898-81741200-dd2b-11e8-961e-a0126c48ba1e.png)

We can modify how `marked` renders the content using the `renderer` helpers: https://marked.js.org/#/USING_PRO.md.

What do you think @prydonius @lentzi90 ?

**Comment by prydonius** [comment]
@Angelmmiguel to clarify, are you suggesting having the table always be expanded, or to have a button next to it to expand it? I think the latter could be very useful, the former might end up hiding other elements.

**Comment by Angelmmiguel** [comment]
@prydonius yea, I'm talking about adding a button to expand the table (toggle). The button will be showed when users' mouse is hover the table. This button will toggle the `overflow` property for the table, so we avoid weird issues with tables that overrides other sections by default.

To clarify the table is expanded and avoid issues with the right sidebar, I'd add a high `z-index` and a `box-shadow` to the table when it's expanded.

**Comment by lentzi90** [comment]
This sounds good to me as well! :+1:
